examples/ddp2pipe/.gitignore

```diff
@@ -0,0 +1 @@
+data
```

Comment:
Is this a folder you used locally, or a folder generated by this example?

Reply:
generated by example for MNIST dataset